### PR TITLE
Fixed: Various Translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- Tests
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes Missing Translates
* Fixes https://github.com/Lidarr/Lidarr/issues/3058
* Fixes https://github.com/Lidarr/Lidarr/issues/2889

### CODE REVIEW

File: frontend/src/Search/AddNewItem.js [hunk 1]
```diff
@@ -166,7 +166,9 @@ class AddNewItem extends Component {
           {
             !isFetching && !error && !items.length && !!term &&
               <div className={styles.message}>
-                <div className={styles.noResults}>Couldn't find any results for '{term}'</div>
+                <div className={styles.noResults}>
+                  {translate('CouldntFindAnyResultsForTerm', [term])}
+                </div>
                 <div>
                   You can also search using the
```
Comment (Qstick) [inline frontend/src/Search/AddNewItem.js:173]: How about this?

Comment (bakerboy448) [reply frontend/src/Search/AddNewItem.js:173]: updated & pushed for both

Note: dropped the links to MB - didn't seem too useful IMO

**Comment by bakerboy448** [comment]
@Qstick any plans to land this xD?

File: frontend/src/AddArtist/ArtistMonitorNewItemsOptionsPopoverContent.js [hunk 2]
```diff
@@ -8,17 +8,17 @@ function ArtistMonitorNewItemsOptionsPopoverContent() {
...
       <DescriptionListItem
         title={translate('NewAlbums')}
-        data="Monitor new albums released after the newest existing album"
+        data={translate('NewAlbumsData')}
       />
 
       <DescriptionListItem
         title={translate('None')}
-        data="Don't monitor any new albums"
+        data={translate('NoneData')}
```
Comment (Qstick) [inline frontend/src/AddArtist/ArtistMonitorNewItemsOptionsPopoverContent.js:21]: Add `Monitor` in front of these tags

**Review by Qstick — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
- rebased
- monitor prefix added as requested
- additional missing translates added

**Comment by Qstick** [comment]
Needs a rebase

**Comment by bakerboy448** [comment]
No plans to rebase

**Comment by Qstick** [comment]
That wasn’t for you, you identified when you closed it originally that you didn’t want anything to do with it

**Pull request closed without merging**